backend/src/main/java/com/cocktailmasters/backend/season/controller/SeasonController.java

```diff
@@ -0,0 +1,86 @@
+package com.cocktailmasters.backend.season.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.cocktailmasters.backend.season.controller.dto.SeasonDto;
+import com.cocktailmasters.backend.season.domain.entity.Season;
+import com.cocktailmasters.backend.season.service.SeasonService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+
+@RestController
+@RequestMapping("/season")
+@Tag(name = "시즌", description = "시즌과 관련된 기능")
+public class SeasonController {
+
+    @Autowired
+    SeasonService seasonService;
+
+    @GetMapping
+    @Operation(summary = "시즌 정보 조회",
+        description = "현재 시즌의 정보(겹치는 시즌이 여러 개 일 경우 먼저 시작한 첫번째 시즌) 또는 지금까지 모든 시즌의 정보 반환")
```

Comment:
코드 컨벤션 순서 변경
어노테이션은 중요도가 높은 것을 아래로
```suggestion
    @Operation(summary = "시즌 정보 조회",
        description = "현재 시즌의 정보(겹치는 시즌이 여러 개 일 경우 먼저 시작한 첫번째 시즌) 또는 지금까지 모든 시즌의 정보 반환")
    @GetMapping
```

Reply:
반영